src/chef-mover/test/mover_eredis_sup_test.erl

```diff
@@ -12,5 +12,5 @@
 %% the start_link path from being followed.
 start_link_should_start_client() ->
     process_flag(trap_exit, true),
-    ?assertEqual({error, {connection_error, {connection_error,nxdomain}}}, mover_eredis_sup:eredis_start_link(host, port)).
-            
+    ?assertEqual({error, {connection_error, {connection_error,timeout}}}, mover_eredis_sup:eredis_start_link("host", 1234)).
```

Comment:
This change fails travis. You had mentioned earlier that it was related to a new version of the `eredis` library, but I don't see a new version getting pulled in here.